scripts/bench_python_cpp_algos.py

```diff
@@ -416,10 +480,12 @@ def load_graph(graph_path, edge_properties=None):
                 print(f"{routine_to_run.path.warn_prefix} {main_warn} {graph_path}")
 
             if curr_edge_load != routine_to_run.edge_load:
+                start_load = time.time()
                 if routine_to_run.edge_load:
                     graph = load_graph(graph_path, [])
                 else:
                     graph = load_graph(graph_path)
+                load_time = time.time() - start_load
```

Comment:
Can this also be a `time_block`? You can just grab the result from the dict or something. (This isn't using the right timer either.)